src/constants/env.js

```diff
@@ -0,0 +1,9 @@
+export const ENV_NAMES = {
+	DEVELOPMENT: 'development',
+	TEST: 'test',
+	PRODUCTION: 'production',
+}
+
+export default {
```

Comment:
As discussed in the issue (but for some reason not showing in this particular interface, possibly GitHub goofiness) let's purge this default export and add the eslint-disabling boilerplate atop this file.